### PR TITLE
L10n context

### PR DESCRIPTION
**Pull request by cbosdo**
## What does this PR change?

* Add `PL()` function to provide context for translators.
* Remove the trailing `: %s` for errors from the localized strings.
* Ensure we don't have multiple unnamed placeholders in strings by adding a position flag.

## Test coverage

- No tests: added a Github action to detect the multiple unnamed placeholders.

- [X] **DONE**

## Links

Issue(s): #

- [X] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

# Before you merge

Check [How to branch and merge properly](https://github.com/uyuni-project/uyuni/wiki/How-to-branch-and-merge-properly)!

### CODE REVIEW

File: shared/api/api.go [hunk 1]
```diff
@@ -109,7 +109,7 @@ func (c *HTTPClient) sendRequest(req *http.Request) (*http.Response, error) {
 
 	res, err := c.Client.Do(req)
 	if err != nil {
-		log.Trace().Msgf("Request failed: %s", err)
+		log.Trace().Err(err).Msgf("Request failed")
```
Comment (mbussolotto) [inline shared/api/api.go:112]: Missing localization.

File: shared/podman/images.go [hunk 2]
```diff
@@ -111,7 +112,7 @@ func GetRpmImagePath(image string) string {
 
 	files, err := os.ReadDir(rpmImageDir)
 	if err != nil {
-		log.Debug().Msgf("Cannot read directory %s: %s", rpmImageDir, err)
+		log.Debug().Err(err).Msgf("Cannot read directory %s", rpmImageDir)
```
Comment (mbussolotto) [inline shared/podman/images.go:115]: Also here and below localization is missing. Maybe regex in `check_localizable` might be improved to find them

**Review by mbussolotto — APPROVED** [review]
just two small nitpick:
- localization is missing for some error message: I think regex in `check_localizable` are not catching them
- it's not completely clear for me when use L(), NL() or PL(). Maybe you can add a comment (or a paragraph in README.md) highlighting when to use one and when another (maybe with an example)

File: shared/podman/images.go (hunk 2 above)
Comment (cbosdo) [reply shared/podman/images.go:115]: Again it's a debug: not localized

**Comment by cbosdo** [comment]
> just two small nitpick:
> 
>     * localization is missing for some error message: I think regex in `check_localizable` are not catching them

I even added a check to not localized debug and traces there!

>     * it's not completely clear for me when use L(), NL() or PL(). Maybe you can add a comment (or a paragraph in README.md) highlighting when to use one and when another (maybe with an example)

Right! the idea is that `L()` is the simple one, `NL()` should be used where there is a plural form of the message and `PL()` is like `L()` with some context for translators. Remember that they don't read the code 😜

**Pull request merged**